### PR TITLE
Feature/kc t 8/support primitives

### PR DESCRIPTION
**Pull request by Lacritz**

### CODE REVIEW

File: processor-kotlin/build.gradle.kts [hunk 1]
```diff
@@ -11,14 +11,21 @@ tasks.withType<Test> {
 }
 
 dependencies {
+    val jacksonVersion = "2.12.2"
```
Comment (justjanne) [inline processor-kotlin/build.gradle.kts:14]: Do we want to define dependencies in the build.gradle.kts, or do we want to define them via gradle.properties (accessing via `val jacksonVersion: String by project`)? Not necessarily something that needs to be changed, but we should keep this in mind for the future.

File: processor-kotlin/src/test/resources/schema/EnumItem.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "types" : [ {
+    "name" : "Status",
+    "values" : [ "RUNNING", "PAUSED", "STOPPED" ],
+    "type" : "ENUM"
+  } ]
+}
```
Comment (justjanne) [inline processor-kotlin/src/test/resources/schema/EnumItem.json]: We should always end files with a line feed.
```suggestion
}

```

**Review by justjanne — CHANGES_REQUESTED** [review]
I've got only minor remarks, otherwise it looks great 👍

File: processor-kotlin/src/test/resources/schema/EnumItem.json (hunk 2 above)
Comment (Lacritz) [reply processor-kotlin/src/test/resources/schema/EnumItem.json]: Agreed (usually), but those files are being used to compare them against compile outputs.. the Compiler output does not have a￼n empty line at the end.

I guess we should rather add a specific compare(file,file) function which just ignores empty lines for comparison. (Unfortunately haven't found any library with such a built-in function)

File: processor-kotlin/build.gradle.kts (hunk 1 above)
Comment (Lacritz) [reply processor-kotlin/build.gradle.kts:14]: Honestly... idc as both ways are just fine. And those are not necessarily advantages over each other

Comment (justjanne) [reply processor-kotlin/build.gradle.kts:14]: In that case, let's stick with what we've got right now :)

**Pull request merged**